### PR TITLE
New updated ing without node_modules

### PR DESCRIPTION
**Pull request by catherine-tao**

### CODE REVIEW

File: frontend/src/Choose.js [hunk 1]
```diff
@@ -16,9 +16,9 @@ const recipelst = [
     {title: "Pizza", 
     description: 'Tastes like newspaper pizza, try it for yourself!', 
     body: 'lorem',
-    image: pizza_img, id:3} 
-]; 
-
+    image: pizza_img, id:3}   
+];            
+//hi
```
Comment (MoonArchitect) [inline frontend/src/Choose.js]: Hi. (that might cause unnecessary merge conflict, unless it is needed there :)

Comment (catherine-tao) [reply frontend/src/Choose.js]: thanks for noticing that! I just took it out :)

**Comment by MoonArchitect** [comment]
Should there be a new `package-lock.json` in the root directory?

**Review by bilalc3 — APPROVED** [review]
looks good

**Review by MoonArchitect — APPROVED** [review]
lgtm

**Pull request merged**